addons/common/functions/fnc_hasItem.sqf

```diff
@@ -16,6 +17,18 @@
  * Public: Yes
  */
 
-params [["_unit", objNull, [objNull]], ["_item", "", [""]]];
+params [["_unit", objNull, [objNull]], ["_item", "", ["", [""]]], ["_checkAll", false, [false]]];
 
-_item in (_unit call EFUNC(common,uniqueItems))
+if (_item isEqualType "") then {
+    _item = [_item];
```

Comment:
could improve performance by just using this as a shorthand to do a single check in _unitItems
instead of creating extra array here and going through the forEach and _checkAll handling below.